### PR TITLE
Add docker-py to core set of docker states

### PR DESCRIPTION
**Pull request by cdarwin**
Fix saltstack-formulas/docker-formula#36

### CODE REVIEW

File: docker/init.sls [hunk 1]
```diff
@@ -66,5 +66,17 @@ docker-service:
     - enable: True
     {% if pkg and "process_signature" in pkg %}
     - sig: {{ pkg.process_signature }}
-    {% endif -%}
+    {% endif %}
 
+python-pip:
```
Comment (iggy) [inline docker/init.sls]: This is almost surely going to cause a id conflict. Please use a more unique id and the name option.

File: pillar.example [hunk 2]
```diff
@@ -10,3 +10,4 @@ docker-pkg:
     version: 1.6.2
     refresh_repo: false
     process_signature: /usr/bin/docker
+    pip_version: '<= 1.2.3'
```
Comment (iggy) [inline pillar.example]: Comment this line and add a comment what this is good for. Too many people will just copy and paste pillar.example files and end up with weird versions.

**Comment by cdarwin** [comment]
@iggy I hope I have addressed your two concerns.

**Comment by iggy** [comment]
Hopefully someone updates that version incompatibility at some point.

Thanks for the submission.

**Pull request merged**